### ISSUE TITLE
Already fixed problems is not available on crates.io

### ISSUE THREAD

**Issue by Fenex** [ecstatic-morse/enum-utils#1]
Last available version of the repository on crates.io is 0.1.1, however the building of the version is terminated with compilation error. Currently [master](https://github.com/ecstatic-morse/enum-utils/tree/f7418cfacf5f4b69ffecdad3b506a2b12904e13f) branch is ok. Could you to update package on crates.io?

I tested on two different platforms and the result is the same:
 * stable-x86_64-unknown-linux-gnu (default)
   rustc 1.38.0 (625451e37 2019-09-23)

 * nightly-armv7-unknown-linux-gnueabihf (default)
   rustc 1.40.0-nightly (6767d9b90 2019-10-11)

P.S. It would be nice to push git tags for all releases.

**Comment by ecstatic-morse** [comment]
I'm very sorry @Fenex.

I believe this was caused by the published version of the subcrate `enum-utils-from-str` getting out of sync with the published version of `enum-utils`. I use a `path` dependency for this, so I did not encounter the problem when building locally or on CI. I will research what the best practices are here so it does not happen again.

In the meantime, I have published `v0.1.2` which should fix the issue. I also published git tags for all published releases (I had them locally but forgot to push). I will try automating this with `cargo-release`.
